### ISSUE TITLE
how to send direct command to ev3 brick

### ISSUE THREAD

**Issue by sugarfree78** [ev3dev/ev3dev-lang-python#644]
I am new to ev3 python but am working on a project. I was reading about NXT-python that " It works by sending the direct commands detailed in LEGO's "Blue-tooth Development Kit" to the NXT brick" I was wondering is it possible to do the same with ev3 brick.please answer in layman's terms.I am using ev3 dev i was hoping i could use it drive robot. more the commands in NXT-python seem to be very simple as compared to the byte commands i have seen please guide regarding this

**Comment by dwalton76** [comment]
See #677 and #699
